libraries/c_sdk/standard/ble/src/services/mqtt_ble/iot_ble_mqtt_transport.c

```diff
@@ -168,24 +162,24 @@ static uint16_t calculateNumSubs( const uint8_t * buf,
  * @param[in] buf, which points to the MQTT packet.
  * @param[out] pIdentifier, which points to packet identifier.
  * @param[in] subscribe, true iff you are parsing a subscribe packet, false if unsubscribe.
- * @return MQTTSuccess or MQTTBadParameter
+ * @return MQTTBLESuccess or MQTTBLEBadParameter
  */
-static MQTTStatus_t parseSubscribe( size_t * subscriptionCount,
-                                    const uint8_t * buf,
-                                    uint16_t * pIdentifier,
-                                    bool subscribe );
+static MQTTBLEStatus_t parseSubscribe( size_t * subscriptionCount,
+                                       const uint8_t * buf,
+                                       uint16_t * pIdentifier,
+                                       bool subscribe );
 
 /**
  * @brief Deserializes a Publish packet in MQTT Format
  * @param[in]
  *
- * statis MQTTStatus_t */
+ * statis MQTTBLEStatus_t */
```

Comment:
Should this comment section be removed?
```suggestion
 * static MQTTBLEStatus_t */
```

Reply:
Removed this comment.